docs/README.md

```diff
@@ -157,6 +157,25 @@ aws_access_key_id = <YOUR ACCESS KEY ID>
 aws_secret_access_key = <YOUR SECRET ACCESS KEY>
 ```
 
+##### Multiple AWS Accounts
+
+In case you are using multiple AWS accounts and not depending on the default profile, you will need to add a `region` field under the profile we will be using in the `credentials` file as seen on the right.
+
+```ini
+# ~/.aws/credentials
+[default]
+aws_access_key_id = <YOUR ACCESS KEY ID>
+aws_secret_access_key = <YOUR SECRET ACCESS KEY>
+region=<Region you want to deploy the application to>
+
+[other_profile]
+aws_access_key_id = <YOUR ACCESS KEY ID>
+aws_secret_access_key = <YOUR SECRET ACCESS KEY>
+region=<Region you want to deploy the application to>
+```
+
+AWS allows for a `config` file to be created, defining the region to deploy to, but this file will only work for the default profile.
+
```

Comment:
I'm not completely sure this is true. I have these 2 files:

```
# ~/.aws/config
[profile booster]
region=eu-west-3
output=table

# ~/.aws/credentials
[booster]
aws_access_key_id=foo
aws_secret_access_key=bar
```

And it's working for me. I have to `export AWS_PROFILE=booster` before running my `boost deploy` or `boost nuke` command

Reply:
Maybe it's because of the keyword `profile` in `[profile booster]`. When I was doing tests I was trying to have it as
```
# ~/.aws/config
[default]
region=us-west-1

[booster]
region=us-west-1
```

and it wasn't working for me. I'll try it out once more with your config and see if it works for me, if it does, then I'll have to update the block once again.

I'll try and remove the unrelated commits now, thanks

Reply:
You were right @itrion, I'll update the text to show users both ways of using the profiles and letting them know they have to export the profile in order for it to work.